Automattic/crowdsignal-plugin
# Add/better template render

#### Changes proposed in this Pull Request:

* Migrate new dashboard looks on wp-admin area
* Remove deprecated multi-user feature
* Raise minimum WP version to 5.5

#### Testing instructions:

* checkout on a self-hosted environment, go to Feedback->Polls and check the dashboard list
* test on different scenarios, with/without multiple users enabled
* create a post with a Crowdsignal Forms block to see the extra tab on the dashboard list
